CONTRIBUTING.md

```diff
@@ -115,9 +115,10 @@ To put forward a PR, we assume you have registered a GitHub ID. Then you could f
 
    ```shell
    golangci-lint run -c .golangci.yml # lint
+   git add -A
    git commit -a -s -m "message for your changes" # -a is git add ., -s adds a Signed-off-by trailer
    git rebase -i	<commit-id> # do this if your pr has multiple commits
-   git push # push to your forked repository after rebase done
+   git push --set-upstream origin <new-branch> # push to your forked repository after rebase done
```

Comment:
seems only first time needs set-upstream?

Reply:
OK, Then I'll delete this, and just add `git add -A`, it's ok?

Reply:
OK, or add some describe